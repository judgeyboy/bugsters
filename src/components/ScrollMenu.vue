<template>
  <ul class="nav" v-scroll-menu>
    <slot></slot>
  </ul>
</template>

<script>
export default {
  name: 'ScrollMenu'
}
</script>

