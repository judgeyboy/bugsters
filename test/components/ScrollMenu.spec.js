import Vue from 'vue'
import SectionScroll from '@/plugins/section-scroll'
import {shallow} from 'vue-test-utils'

import ScrollMenu from '@/components/ScrollMenu'

Vue.use(SectionScroll)

describe('ScrollMenu component', () => {
  it('should be a vue instance', () => {
    const wrapper = shallow(ScrollMenu)
    expect(wrapper.isVueInstance()).toBe(true)
  })

  it('should match snapshot', () => {
    const wrapper = shallow(ScrollMenu)
    const $html = wrapper.vm.$el.outerHTML
    expect($html).toMatchSnapshot()
  })
})
