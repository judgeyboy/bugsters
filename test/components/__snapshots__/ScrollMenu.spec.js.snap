// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollMenu component should match snapshot 1`] = `"<ul class=\\"nav\\"></ul>"`;
